### PR TITLE
[Bugfix] Refactor transactions and missing details #172

### PR DESCRIPTION
**Pull request by kcw-grunt**
## Problem/Issue
 The Transactions / History View has the transactions where the user can tap on the detail.  But the detail sometimes disappear.

 ## Approach
 There are a few ways of fixing the problem. One would be a patch but it meant the poor architecture would remain. This was dismissed immediately because there is plan to migrate to SwiftUI (_from UIKit_) for all cases.

 The reason this work was avoided was because the amount of refactoring and developing was significant (**8 pts +**). This took 1 week and while the fix was put in, any related bugfixes and enhancements were incorporated

 Some steps:
 - Change the transaction view cell to a SwiftUI paradigm
 - Layout with the new cell
 - Add Model and Modal View
 - Any related bugfixes 

 ## Screenshots

 Before | After
 -------|-------
 buggytransactionclip|transactionclip

### CODE REVIEW

File: loafwallet/PromptTableViewCell.swift [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    
+    @IBAction func didTapAction(_ sender: Any) {
+        didTap?()
+    }
+    
+    @IBAction func closeAction(_ sender: Any) {
+        didClose?()
+    }
+    
+    override func awakeFromNib() {
```
Comment (mosadialiou) [inline loafwallet/PromptTableViewCell.swift:30]: No need to override this function. Can be removed!

File: loafwallet/PromptTableViewCell.swift [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        super.awakeFromNib()
+    }
+    
+    override func setSelected(_ selected: Bool, animated: Bool) {
```
Comment (mosadialiou) [inline loafwallet/PromptTableViewCell.swift:34]: No need to override this function. Can be removed!

File: loafwallet/TransactionCellView.swift [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+       
+    init(viewModel: TransactionCellViewModel) {
+        self.viewModel = viewModel
+    }
+    
+    var body: some View {
+
+        VStack(alignment: .leading, spacing: 1.0) {
+                
+           // Spacer(minLength: 2.0)
```
Comment (mosadialiou) [inline loafwallet/TransactionCellView.swift:29]: commented code to remove!

**Review by mosadialiou — CHANGES_REQUESTED** [review]

File: loafwallet/PromptTableViewCell.swift (hunk 2 above)
Comment (kcw-grunt) [reply loafwallet/PromptTableViewCell.swift:34]: its needed for setSelected

**Comment by kcw-grunt** [comment]
cc: @jeff169

**Review by mosadialiou — APPROVED** [review]
LGTM

**Pull request merged**